lib/service_check.pm

```diff
@@ -195,6 +202,10 @@ sub _is_applicable {
         my %included = map { $_ => 1 } split(/\s*,\s*/, get_var('INCLUDE_SERVICES'));
         return 0 unless ($included{$srv_pkg_name});
     }
+    if ($srv_pkg_name eq 'hpcpackage_remain' && !check_var('SLE_PRODUCT', 'hpc')) {
```

Comment:
The logic should move to line 196 since we give exclude/include setting more priority